src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamDay.java

```diff
@@ -122,9 +137,12 @@ public EventStreamDay copy() {
             EventStreamWindow newWindow = new EventStreamWindow();
             newWindow.setTimeWindowGuid(win.getTimeWindowGuid());
             newWindow.setState(win.getState());
+            newWindow.setStartDate(win.getStartDate());
+            newWindow.setStartTime(win.getStartTime());
             newWindow.setSessionInstanceGuid(win.getSessionInstanceGuid());
             newWindow.setEndDay(win.getEndDay());
             newWindow.setEndDate(win.getEndDate());
+            newWindow.setEndTime(win.getEndTime());
```

Comment:
Do we need to update the copy() unit test?

Reply:
Updated